useful-scripts/misc-test-utils/src/services/commonService.ts

```diff
@@ -0,0 +1,202 @@
+      const count = await zip.extract(null, `${cwd}/test-files`);
+      console.log(chalk.green(`Extracted ${count} entries to test-files`));
+      await zip.close();
+    }
+  },
+  renameFiles: async () => {
+    // define a random id to assign to the renamed data & meta file...
+    const randomId = Math.floor(Math.random() * 1000000);
+
+    // rename csv file..
+    const csvGlob1 = await globby(`${cwd}/test-files/*.csv`);
+
+    const csvGlob2 = csvGlob1[0];
+
+    fs.rename(`${csvGlob2}`, `${cwd}/test-files/testfile-${randomId}.csv`, e =>
+      e ? console.error(chalk.red(e)) : '',
+    );
+
+    // rename the meta file...
+    const metaGlob = await globby(`${cwd}/test-files/*.meta.csv`);
+    const metaGlob2 = metaGlob[0];
+
+    fs.rename(
+      `${metaGlob2}`,
+      `${cwd}/test-files/testfile-${randomId}.meta.csv`,
+      e => (e ? console.error(chalk.red(e)) : ''),
+    );
+  },
+
+  addMetaGuidance: async (
+    subjArr: { id: string; content: string }[],
+    releaseId: string,
+  ): Promise<boolean> => {
+    try {
+      await axios({
+        method: 'PATCH',
+        url: `${ADMIN_URL}/api/release/${releaseId}/meta-guidance`,
+        headers: {
+          Authorization: `Bearer ${JWT_TOKEN}`,
+          'Content-Type': 'application/json',
+        },
+        data: {
+          content: '<p>testing</p>',
+          subjects: subjArr,
+        },
+      });
+      return true;
+    } catch (e) {
+      errorHandler(e);
+    }
+    return false;
+  },
+
+  getFinalReleaseDetails: async (releaseId: string) => {
+    try {
+      const res = await axios({
+        method: 'GET',
+        url: `${ADMIN_URL}/api/releases/${releaseId}`,
+        headers: {
+          Authorization: `Bearer ${JWT_TOKEN}`,
+          'Content-Type': 'application/json',
+        },
+      });
+      const releaseData: ReleaseData = res.data;
+
+      await sleep(1000);
+
+      return {
+        id: releaseData.id,
+        title: releaseData.title,
+        slug: releaseData.slug,
+        publicationId: releaseData.publicationId,
+        publicationTitle: releaseData.publicationTitle,
+        publicationSlug: releaseData.publicationSlug,
+        releaseName: releaseData.releaseName,
+        yearTitle: releaseData.yearTitle,
+        typeId: releaseData.typeId,
+        live: 'false',
+        timePeriodCoverage: {
+          value: releaseData.timePeriodCoverage.value,
+          label: releaseData.timePeriodCoverage.label,
+        },
+        preReleaseAccessList: '',
+        latestRelease: 'false',
+        type: {
+          id: releaseData.type.id,
+          title: releaseData.type.title,
+        },
+        contact: {
+          id: releaseData.contact.id,
+          teamName: releaseData.contact.teamName,
+          teamEmail: releaseData.contact.teamEmail,
+          contactName: releaseData.contact.contactName,
+          contactTelNo: releaseData.contact.contactTelNo,
+        },
```

Comment:
We could simplify this by spreading in the `releaseData` e.g.

```ts
return {
  ...releaseData,
  approvalStatus: 'Approved',
  // ... the things you're overriding it with 
};
```

I'm not entirely sure why we're overriding specific things though? Shouldn't we just return the release? In which case, this method could also just be named `getRelease`